<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupControls">
     <property name="title">
      <string>Controls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="labelDistribution">
          <property name="text">
           <string>Distribution</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="inputDistribution">
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelN">
          <property name="text">
           <string>Samples</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="inputN">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelMean">
          <property name="text">
           <string>Mean:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="valueMean">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelStd">
          <property name="text">
           <string>Stand. dev.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="valueStd">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCustomDistribution">
        <property name="text">
         <string>1/(1 * sqrt(2 * pi)) * exp(-0.5 * ((x - 0) / 1)^2)</string>
        </property>
        <property name="placeholderText">
         <string>Input distribution expression</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="valueError">
        <property name="text">
         <string>No errors</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>3</number>
     </property>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Sample</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Distribution</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>-1</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSample">
     <property name="title">
      <string>Sample</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonImport">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExport">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
